exe/3scale-copy

```diff
@@ -9,12 +9,12 @@ options = {}
 parser = OptionParser.new do |parser|
   parser.banner = '3scale copy <command> [options]'
 
-  parser.on('-e', '--endpoint ENDPOINT', "Endpoint") do |domain|
-    options[:endpoint] = domain
+  parser.on('-source', '--source SOURCE', "Source") do |domain|
```

Comment:
why changing it from `-e` to `-source` ? I'd understand `--endpoint` to `--source`. but why the one dash shortcut to have full word? or is optparser smart and shows just first letter?